liquibase-standard/src/main/java/liquibase/command/core/HistoryCommand.java

```diff
@@ -12,7 +12,7 @@
 /**
  * @deprecated Implement commands with {@link liquibase.command.CommandStep} and call them with {@link liquibase.command.CommandFactory#getCommandDefinition(String...)}.
  */
-public class HistoryCommand extends AbstractCommand {
+public class HistoryCommand extends AbstractCommand<CommandResult> {
```

Comment:
Not sure if this could have a negative impact, but we try to avoid updating public elements (return types, method signatures, class definition, etc) as in some cases it could be a breaking change for some people depending on how they are using methods/classes. Maybe this case doesn't hurt, but considering our guidelines I should revert this one.